rules/import-style.js

```diff
@@ -138,6 +138,7 @@ const create = context => {
 			checkDynamicImport = true,
 			checkExportFrom = false,
 			checkRequire = true,
+			autoFix = false,
```

Comment:
We don't need option for this, it should always do autofix.

Reply:
Got it. When I ran tests there was an error that told me autofixes were unexpected. This is my first time submitting a PR here, so I can't quite remember what caused that to happen, but I can backtrack and fix it.